lib/jsdom/living/traversal/NodeFilter-impl.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+const { addConstants } = require("../utils");
+
+module.exports = function (core) {
```

Comment:
This file should not be here. After https://github.com/jsdom/jsdom/commit/87fcce674447d0468285f3ef70f068f274d1b36a, the file will be automatically generated by webidl2js.

Make sure your local dependencies are up-to-date.